lib/util.sh

```diff
@@ -9,14 +9,20 @@ set -o pipefail
 # Globals:
 #   ARGS
 #   FILES
+#   NO_CD
 #######################################
 function util::parse_cmdline() {
   # Global variable arrays
   ARGS=()
   FILES=()
+  export NO_CD=0
```

Comment:
`export` is needed to silence shellcheck [SC2034](https://www.shellcheck.net/wiki/SC2034).
It works without it, but shellcheck complains:
```
shellcheck...............................................................Failed
- hook id: shellcheck
- exit code: 1

In lib/util.sh line 23:
        NO_CD=1
        ^---^ SC2034 (warning): NO_CD appears unused. Verify use (or export if used externally).

```